src/Command/EnqueueCommand.php

```diff
@@ -4,17 +4,19 @@
 
 namespace Webgriffe\SyliusAkeneoPlugin\Command;
 
-use Sylius\Component\Resource\Factory\FactoryInterface;
+use DateTime;
+use InvalidArgumentException;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Command\LockableTrait;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Throwable;
 use Webgriffe\SyliusAkeneoPlugin\DateTimeBuilderInterface;
-use Webgriffe\SyliusAkeneoPlugin\Entity\QueueItemInterface;
 use Webgriffe\SyliusAkeneoPlugin\ImporterInterface;
 use Webgriffe\SyliusAkeneoPlugin\ImporterRegistryInterface;
-use Webgriffe\SyliusAkeneoPlugin\Repository\QueueItemRepositoryInterface;
+use Webgriffe\SyliusAkeneoPlugin\Message\ItemImport;
 use Webmozart\Assert\Assert;
 
 final class EnqueueCommand extends Command
```

Comment:
A questo punto cambierei anche il nome di questo comando in quanto il fatto che metta solo in coda è un dettaglio implementativo trasparente per l'utente. Tra l'altro il messenger credo si possa configurare anche in maniera sincrona...